pkg/controllers/capr/unmanaged/controller.go

```diff
@@ -163,7 +163,7 @@ func (h *handler) createMachineObjects(capiCluster *capi.Cluster, machineName st
 
 	labels[capr.MachineIDLabel] = data.String("id")
 	labels[capr.ClusterNameLabel] = capiCluster.Name
-	labels[capi.ClusterLabelName] = capiCluster.Name
+	labels[capi.ClusterNameLabel] = capiCluster.Name
```

Comment:
Duplicate

Reply:
Hi, capr.ClusterNameLabel and capi.ClusterNameLabel are different. I don't think its duplicate.

Reply:
It is a duplicate of the line directly above

Reply:
No problem. :)